### ISSUE TITLE
Error in ip_cliente.py

### ISSUE THREAD

**Issue by daniguille99** [gisce/iec870ree#39]
Hello!
I'm having an issue when I run ip_cliente.py with my meter. It seems to work well, but at some point, it appears the next error:

![Captura de pantalla 2022-11-14 100456](https://user-images.githubusercontent.com/62031316/201619262-063a5445-7afa-4e71-81fa-33f57dd88df2.png)

Does anyone now what that error exactly means and how can I solve it?

Thank you in advance!

**Comment by tinogis** [comment]
A Bad ASDU is received. The length field of the ASDU is not correct. Is possible that a non iec870ree paquet is received?

**Comment by daniguille99** [comment]
I don't think so, because the protocol used by the meter is iec870ree. What I did was to run the ip_client.py in vscode only changing the information of the meter and while at first seems to work at this point the error appears:
![image](https://user-images.githubusercontent.com/62031316/201684386-8788a20d-a68c-42e7-bc52-4c0789c58ceb.png)
I also tried to change the datetime but the error keeps appearing.
Maybe I misread the meter values, which i interpreted in that way:
der: "Dirección de enlace"
dir_pm: "Punto de Medida"
pwd: "Password"
